.github/workflows/build-assets.yml

```diff
@@ -170,13 +173,14 @@ jobs:
         if: ${{ matrix.container }}
         run: |
           mkdir -p ${{ needs.release.outputs.folder_build }}
+          echo "${{ secrets.SIGNING_SECRET }}" > ${{ needs.release.outputs.folder_build }}/private.key
           sudo rm -f ${{ needs.release.outputs.folder_build }}/CMakeCache.txt
           sudo rm -rf ${{ needs.release.outputs.folder_build }}/CMakeFiles
           docker_label=${{ secrets.DOCKER_USERNAME }}/build-${{ matrix.os }}:${{ needs.release.outputs.community_tag }}
-          docker run --rm --mount ${{ needs.release.outputs.mount_platform }} --mount ${{ needs.release.outputs.mount_build }} $docker_label "cmake -S /hpcc-dev/HPCC-Platform -B /hpcc-dev/build ${{ needs.release.outputs.cmake_options }} -DINCLUDE_PLUGINS=ON -DCONTAINERIZED=ON -DSUPPRESS_REMBED=ON -DSUPPRESS_V8EMBED=ON -DSUPPRESS_SPARK=ON -DCPACK_STRIP_FILES=OFF"
-          docker run --rm --mount ${{ needs.release.outputs.mount_platform }} --mount ${{ needs.release.outputs.mount_build }} $docker_label "cmake --build /hpcc-dev/build --parallel $(nproc) --target package" 
-          docker run --rm --mount ${{ needs.release.outputs.mount_platform }} --mount ${{ needs.release.outputs.mount_build }} $docker_label "cmake -S /hpcc-dev/HPCC-Platform -B /hpcc-dev/build ${{ needs.release.outputs.cmake_options }} -DINCLUDE_PLUGINS=ON -DCONTAINERIZED=ON -DSUPPRESS_REMBED=ON -DSUPPRESS_V8EMBED=ON -DSUPPRESS_SPARK=ON -DCPACK_STRIP_FILES=ON" 
-          docker run --rm --mount ${{ needs.release.outputs.mount_platform }} --mount ${{ needs.release.outputs.mount_build }} $docker_label "cmake --build /hpcc-dev/build --parallel $(nproc) --target package"
+          docker run --rm --mount ${{ needs.release.outputs.mount_platform }} --mount ${{ needs.release.outputs.mount_build }} $docker_label "${{ needs.release.outputs.load_key }} && cmake -S /hpcc-dev/HPCC-Platform -B /hpcc-dev/build ${{ needs.release.outputs.cmake_options }} -DSIGN_MODULES_PASSPHRASE=${{ secrets.SIGNING_PASSPHRASE }} -DSIGN_MODULES_KEYID=${{ secrets.SIGNING_KEYID }} -DINCLUDE_PLUGINS=ON -DCONTAINERIZED=ON -DSUPPRESS_REMBED=ON -DSUPPRESS_V8EMBED=ON -DSUPPRESS_SPARK=ON -DCPACK_STRIP_FILES=OFF"
```

Comment:
Any reason not to have -DSIGN_MODULES_PASSPHRASE=${{ secrets.SIGNING_PASSPHRASE }} -DSIGN_MODULES_KEYID=${{ secrets.SIGNING_KEYID }} in the global cmake_options, given SIGN_MODULE is there?

Reply:
There actually is a reason. Because it's a secret, github actions won't allow it to be stored there because it can tell it will be dumped out to GITHUB_OUTPUT and it can't ensure the secret will stay safe.  So we're forced to use it when we need it in the cmake calls.

Reply:
I think the following format would work:
```
//  In vars:
echo ::set-output name=XXX::${{ secrets.XXXX }}
```